### DISCUSSION TITLE
Suit link on EDMC

### DISCUSSION THREAD

**Discussion by Falcon911** [EDCD/EDMarketConnector#1149] [Ideas]
Since we have options on EDMC currently for choose out Ship, System and Station. Would only seem right to have a link for Suits to be pointed to inara (unless there is a second option I am not aware of ) Commander>Inventory and Crafting>Inventory>Ship Locker.  Add in EDMC> Configuration page > Below Shipyard > Suits> Either Off or Inara

**Comment by Athanasius** [comment]
NB: Despite the site navigation ending up on a URL that contains the user's ID, this also works: https://inara.cz/cmdr-inventory/#tab_commanderinventoryshiplocker

**Reply by Athanasius** [reply]
And there's also https://inara.cz/cmdr-equipment-loadouts/
